Automattic/woocommerce-payments
# Bump minimum WooCommerce version to 4.5

#### Changes proposed in this Pull Request

Bump the minimum supported version of WooCommerce to 4.5.

#### Testing instructions

Check that CI tests run as expected. These were updated to stop testing versions <4.5.
